ctapipe/image/tests/test_geometry_converter.py

```diff
@@ -12,19 +12,33 @@
 cam_ids = CameraGeometry.get_known_camera_names()
 
 
+def create_mock_image(geom):
```

Comment:
maybe one could expose the "magic numbers" as parameters of this function, like:
```
def create_mock_image(
    geom,
    centroid=0.3,
    width=0.03,
    length=0.10,
    psi="25d",
    intensity=0.5,
    nsb_level_pe=3,
):
```

Reply:
or maybe one could somehow comment on this particular choice for these 6 parameters?

Reply:
The function is defined in the test, so is not a user facing function. The whole does also not work with astropy units. So if we would like to generalize this function, more work is needed anyway.

Reply:
ok